tests/stave_tests.ts

```diff
@@ -690,8 +690,8 @@ function drawTempo(options: TestOptions, contextBuilder: ContextBuilder): void {
   y += 150;
 
   drawTempoStaveBar(120, { duration: 'w', bpm: 80 }, 0);
-  drawTempoStaveBar(100, { duration: 'h', bpm: 90 }, 0);
-  drawTempoStaveBar(100, { duration: 'q', bpm: 96 }, 0);
+  drawTempoStaveBar(100, { duration: 'h', dots2: 1, duration2: 'q' }, 0);
```

Comment:
In the tests / examples, we should probably always put dots after duration for consistency. It looks weird here to have duration / dots2.  It should be duration / duration2 / dots2.

Reply:
done